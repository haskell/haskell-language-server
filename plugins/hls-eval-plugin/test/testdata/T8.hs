-- Semantic and Lexical errors are reported
module T8 where

-- >>> noFunctionWithThisName

-- >>> "a" + "bc"

-- >>> "

-- >>> 3 `div` 0
