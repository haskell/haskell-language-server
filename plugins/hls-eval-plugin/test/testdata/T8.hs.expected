-- Semantic and Lexical errors are reported
module T8 where

-- >>> noFunctionWithThisName
-- Variable not in scope: noFunctionWithThisName

-- >>> "a" + "bc"
-- No instance for (Num [Char]) arising from a use of '+'

-- >>> "
-- lexical error in string/character literal at end of input

-- >>> 3 `div` 0
-- divide by zero
